boxes/forms.py

```diff
@@ -1,12 +1,26 @@
 from django.forms import ModelForm, Form, CharField, Textarea
 from .models import Box
+from django.utils import timezone
 
 
 class CreateBoxForm(ModelForm):
     class Meta:
         model = Box
         fields = ["name", "description", "expires_at"]
 
+    def clean_expires_at(self):
+        # Validate the expiration date
+        if self.expires_at:
+            expires_at = self.expires_at
+        else:
+            expires_at = self.cleaned_data.get("expires_at", "")
+
+        if expires_at:
+            # Check if the expiration date is a past date
+            if timezone.now() > expires_at:
```

Comment:
@ricardopereira in the line you probably will need to do something like this: 
```python
timezone.localtime(timezone.now(), timezone.get_current_timezone())
```
Sorry I might had mislead you (my fault), but the comparison is only useful if the user timezone is taken into account. `timezone.now()` returns an aware datetime in UTC, so the conversion will be needed.